concepticondata/conceptlists/Speed-2017-485.tsv

```diff
@@ -0,0 +1,486 @@
+Speed-2017-485-43	43	boom	tree	0.9375	0	2.375	2.5	4.3125	zien	0.425925926	2.083333333	5	5.166666667	906	TREE
+Speed-2017-485-44	44	boor	drill	3.866666667	0	0.666666667	1.866666667	3.2	horen	0.402777778	4.916666667	5.083333333	2.666666667	3423	DRILL
+Speed-2017-485-45	45	boot	boat	1.666666667	0	0.6	1.466666667	4	zien	0.517241379	4.5	4.833333333	4.666666667	1844	BOAT
+Speed-2017-485-46	46	boter	butter	0.066666667	4.133333333	1.866666667	1.066666667	3.333333333	proeven	0.388535032	2.916666667	2.916666667	4.333333333	1245	BUTTER
+Speed-2017-485-47	47	bouillon	broth	0	4.733333333	3.4	0.6	2.933333333	proeven	0.405714286	3.666666667	4	4.583333333	1547	SOUP
+Speed-2017-485-48	48	braille	braille	0.066666667	0	0	4.333333333	2.533333333	voelen	0.625	3.833333333	3.583333333	4.583333333		
+Speed-2017-485-49	49	brie	brie	0	4.625	3.6875	2.25	3.6875	proeven	0.324561404	3.5	4.833333333	3.5		
+Speed-2017-485-50	50	bries	breeze	1.785714286	0	0.142857143	3.571428571	0.428571429	voelen	0.602409639	3.25	3.416666667	4.5		
+Speed-2017-485-51	51	brievenbus	mailbox	1.928571429	0	0	2.285714286	4.071428571	zien	0.49137931	3.416666667	2.916666667	4.166666667		
+Speed-2017-485-52	52	bril	glasses	0.0625	0	0	3.0625	4.625	zien	0.596774194	3.333333333	3.916666667	3.666666667	1898	SPECTACLES
+Speed-2017-485-53	53	broccoli	broccoli	0	3.866666667	1.666666667	1.866666667	3.666666667	proeven	0.34939759	3.5	4.083333333	3.916666667	3726	BROCCOLI
+Speed-2017-485-54	54	brood	bread	0.133333333	4.733333333	3.2	1.666666667	3.266666667	proeven	0.353846154	3.583333333	3.833333333	5.416666667	1368	BREAD
+Speed-2017-485-55	55	brug	bridge	0.266666667	0	0	1.4	4.266666667	zien	0.719101124	4.25	5.25	4.666666667	1011	BRIDGE
+Speed-2017-485-56	56	buskruit	gunpowder	2.266666667	0.266666667	1.933333333	0.4	2.466666667	zien	0.3	5.583333333	6.166666667	3.083333333	1798	GUNPOWDER
+Speed-2017-485-57	57	cacao	cocoa	0.133333333	4.066666667	2.533333333	0.333333333	2.4	proeven	0.415492958	4.166666667	4.083333333	5.416666667	2442	COCOA BEAN
+Speed-2017-485-58	58	cactus	cactus	0	0.533333333	0.266666667	3.333333333	4	zien	0.491803279	3.333333333	4.75	3.833333333	2595	CACTUS
+Speed-2017-485-59	59	camera	camera	1.625	0	0	2.9375	4.375	zien	0.48951049	4.583333333	4.666666667	5.083333333	2618	CAMERA
+Speed-2017-485-60	60	cannabis	cannabis	0	1.071428571	3.5	0.428571429	2.142857143	ruiken	0.49	2.583333333	5.083333333	2.333333333		
+Speed-2017-485-61	61	cashew	cashew	0.333333333	4.266666667	1.2	1.8	3.533333333	proeven	0.353293413	3.333333333	3.5	3.916666667	2441	CASHEW
+Speed-2017-485-62	62	ceder	cedar	0	0.5	1.75	0.583333333	2	zien	0.413793103	3.333333333	3.75	3.833333333		
+Speed-2017-485-63	63	chai	chai	0	3.4	3.4	0.9	1.8	joint	0.357894737	3	3.666666667	4.166666667		
+Speed-2017-485-64	64	champignon	mushroom	0	3.933333333	1.866666667	2.066666667	3.8	proeven	0.337142857	3.666666667	3.5	3.916666667		
```

Comment:
For this and the previous we could really need some Dutch help, and for things like that in general.